API/users/models.py

```diff
@@ -1,2 +1,45 @@
+from django.db import models
+from django.contrib.auth.models import User
 
 # Create your models here.
+
+class Account(models.Model):
+    name = models.CharField(max_length=64)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    def __str__(self):
+        return self.name
+
+class Transaction(models.Model):
+    account = models.ForeignKey(Account, on_delete=models.CASCADE)
+    timestamp = models.DateTimeField()
+    def __str__(self):
+        return self.name
```

Comment:
No `name` field in this model

Reply:
I'm removing the `def __str(self)` wherever there isn't a possibility to include one then